db/post_migrate/20200917193528_migrate_notifications_type.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class MigrateNotificationsType < ActiveRecord::Migration[5.2]
+  disable_ddl_transaction!
+
+  TYPES_TO_MIGRATE = %w(
+    Mention
+    Status
+    Status
+    Follow
+    FollowRequest
+    Favourite
+    Poll
+  ).freeze
+
+  def up
+    TYPES_TO_MIGRATE.each do |activity_type|
+      Notification.where(activity_type: activity_type).in_batches.update_all(type: activity_type.underscore)
```

Comment:
`Notification.where(activity_type: activity_type, type: nil)`, otherwise there's a risk to overwrite the new notification type